sdk/formrecognizer/Azure.AI.FormRecognizer/src/autorest.md

```diff
@@ -7,12 +7,10 @@ Run `dotnet build /t:GenerateCode` to generate code.
 
 ``` yaml
 input-file:
-    -  https://github.com/Azure/azure-rest-api-specs/blob/8fee11f5dd30e228a850ee25a2f3368dc1671b53/specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.1/FormRecognizer.json
+    - .\swagger\fr2.1p2.json
```

Comment:
Are we keeping a temporary local swagger while the service team does not update it on their side?

Reply:
yeah... they have a PR open but not much progress so we internally are using this swagger to get unblock and develop new features.
This work is only in feature branch. I will only move to master when the service team merges their PR